libcxx/test/std/utilities/text_encoding/text_encoding.members/text_encoding.aliases_view/begin.pass.cpp

```diff
@@ -0,0 +1,65 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <text_encoding>
+
+// REQUIRES: std-at-least-c++26
+
+// struct text_encoding::aliases_view
+
+#include <cassert>
+#include <ranges>
+#include <text_encoding>
+
+using id = std::text_encoding::id;
+
+constexpr bool tests() {
+  // 1. begin() of an aliases_view from a single text_encoding object are the same.
+  {
+    auto te    = std::text_encoding(id::UTF8);
+    auto view1 = te.aliases();
+    auto view2 = te.aliases();
```

Comment:
AFAIK It is recommended to specify the full type and avoid using `auto` if possible.